src/django/api/serializers.py

```diff
@@ -18,6 +23,41 @@ def create(self, validated_data):
         user.save()
         return user
 
+    def get_name(self, user):
+        try:
+            user_organization = Organization.objects.get(admin=user)
```

Comment:
Do you know if this `Organization.objects.get(admin=user)` result is cached in any way? Are we querying the database 5 times per serialization?

Also, why make the API different from the underlying models? Convenience, or do we expect the underlying models to change again?

Reply:
> why make the API different from the underlying models? Convenience, or do we expect the underlying models to change again?

This was partly convenience and partly because the original client's list of form fields used the `contributor type` and `other contributor type` nomenclature. Changing the variable names would then cast them out of sync with the client's form field labels rather than out of sync with the models. "Contributor" was also the term used on the client's search sidebar; the Organization model is something we introduced in the transition to DRF, but it's currently one-to-one with Users and isn't currently exposed anywhere in the client.

In general I thought it'd remain fairly easy to change it wholesale later if desired.

Good q about the caching: I don't know the answer offhand.

Reply:
I don't have enough context to see which way this project is headed, but in general simplicity is always better. If we can update the client's form fields to match the new variable names, that would be better. But that can also be a follow up task, doesn't necessarily have to be done as part of this.

Reply:
Makes sense to me. I think handling it in a new card would be best; I noticed that the filter reducers also use the `contributor` and `contributorType` variables, so it'd probably be worthwhile to have the changes be made & reviewed wholesale.

Reply:
Made https://github.com/open-apparel-registry/open-apparel-registry/issues/185 for addressing that btw.

Reply:
Great, thanks!